src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -121,6 +121,7 @@ public void AllowReadStreamBuffering_SetTrueThenGet_ExpectTrue(Uri remoteServer)
             Assert.True(request.AllowReadStreamBuffering);
         }
 
+        [OuterLoop]
```

Comment:
This is indeed wrong and should be moved back to InnerLoop again. Just verified it with loops